consensus/src/counters.rs

```diff
@@ -120,6 +117,65 @@ pub static LEADER_REPUTATION_ROUND_HISTORY_SIZE: Lazy<IntGauge> = Lazy::new(|| {
     .unwrap()
 });
 
+/// Computed at leader election time, with some delay.
+
+/// Window sizes for which to measure chain health.
+pub static CHAIN_HEALTH_WINDOW_SIZES: [usize; 3] = [10, 100, 1000];
```

Comment:
Let's also add 1 to the window size, which is going to give us the chain health for the last round?

Reply:
1 is pretty much useless, it is always going to be == 2f+1, not larger not smaller. 

10 looks noisy too, i.e. not all alive nodes get a chance to vote. 100 looks pretty stable.

We might want to change these to [30, 100, 300] later, but this seems fine for now.